sa/model.go

```diff
@@ -682,6 +682,45 @@ func hasMultipleNonPendingChallenges(challenges []*corepb.Challenge) bool {
 	return false
 }
 
+// newAuthzReqToModel converts an sapb.NewAuthzRequest to the authzModel storage
+// representation.
```

Comment:
It also hardcodes the status as pending, which I think is fine, but we should document it as part of the contract because I found it to be a little surprising.

Reply:
Good call, done.